src/styles/main.scss

```
@@ -1 +1,58 @@
-// put code here
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 1vmin dotted #000;
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__minute,
+  &__second,
+  &__center {
+    position: absolute;
+  }
+
+  &__minute {
+    width: 20vmin;
+    height: 3vmin;
+    background-color: #0700ff;
+
+    animation: rotate-arr 3600s steps(60) infinite;
```

Create variable for this 
```suggestion
    animation: rotate-arr ... steps(60) infinite;
```